calling/src/androidMain/kotlin/com/wire/kalium/calling/Platform.kt

```diff
@@ -0,0 +1,5 @@
+package com.wire.kalium.calling
+
+actual class Platform actual constructor() {
+    actual val platform: String = "Android ${android.os.Build.VERSION.SDK_INT}"
+}
```

Comment:
Is a `Platform` class really necessary for this module?

Reply:
No, It's automatically created. 
I just left it to keep the module's structure as it is not possible to only push folders